soweego/commons/logging.py

```diff
@@ -50,7 +50,7 @@
         'debug_file_handler': {
             'formatter': 'soweego',
             'level': 'DEBUG',
-            'filename': 'debug.log',
+            'filename': '/app/shared/debug.log',
```

Comment:
This path is problematic when not working with Docker, see for instance #166

Reply:
Approving this change anyway: since my laptop seems to be the only one hitting #166, I'll manually change the path on my own

Reply:
@marfox I get it. How can we set the log path in a better way? Have you got any suggestion?

Reply:
You can pass a configuration file, see `commons/logging#setup`. It would actually be preferable to do so, otherwise everything will break if running outside Docker